stories/Components/Forms/Dropdowns/Multiselect/multiselect.scss

```diff
@@ -59,128 +73,136 @@
       @extend %arrow;
     }
   }
-}
 
-.options {
-  @extend %select_option;
+  ul {
+    @extend %select_option;
 
-  -ms-overflow-style: -ms-autohiding-scrollbar;
-  overflow-y: auto;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+    overflow-y: auto;
 
-  > li {
-    border-top: 1px solid $color-gray-400;
-    cursor: pointer;
-    list-style: none;
-    margin-bottom: 0;
+    > li {
+      border-top: 1px solid $color-gray-400;
+      cursor: pointer;
+      list-style: none;
+      margin-bottom: 0;
 
-    > .checkbox {
-      &:hover {
-        background-color: $color-gray-100;
+      > .checkbox {
+        &:hover {
+          background-color: $color-gray-100;
+        }
+      }
+      ul {
+        .checkbox {
+          &:hover {
+            background-color: inherit;
+          }
+        }
       }
     }
-  }
 
-  .has-submenu {
-    position: relative;
+    .has-submenu {
+      position: relative;
 
-    &.open-submenu {
-      .sub-menu {
-        border-top: solid 1px $color-gray-400;
-        display: block;
-      }
+      &.open-submenu {
```

Comment:
@Ashish-srij Change class name to just `open` as we already have `.has-submenu` as the main class.

Reply:
@shishirphadke Done.